tests/netstats_l2/tests/01-run.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2017 Alexandre Abadie <alexandre.abadie@inria.fr>
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+import os
+import sys
+
+EXPECTED_HELP = (
+    'Command              Description',
+    '---------------------------------------',
+    'reboot               Reboot the node',
+    'ifconfig             Configure network interfaces',
+    '>'
+)
+
+EXPECTED_IFCONFIG = (
+    r'ifconfig',
+    r'Iface  \d+   HWaddr\: ([a-f0-9]{2}[:-]){5}([a-f0-9]{2}) ',
```

Comment:
The `{5}` is wrong on `samr21-xpro` as it shows a 2 long address, so itcould be replaced by `+`.
Also, the `a-f0-9` could be replaced by `0-9a-f` to sort the hexadecimal characters by value.

`r'Iface  \d+   HWaddr\: ([0-9a-f]{2}[:-])+([0-9a-f]{2}) '`

Question, why do you have `[:-]` and not only `:` ?